src/tropical_bitwise.jl

```diff
@@ -0,0 +1,109 @@
+"""
+    TropicalBitwise{T} <: AbstractSemiring
+
+TropicalBitwise is a semiring algebra, can be described by
+* TropicalBitwise, (ℝ, |, &, 0, ~0).
+
+It maps
+* `+` to `|`
+* `*` to `&`
+* `0` to `0`
+* `1` to ~0`
```

Comment:
```suggestion
* `1` to `~0`
```

Reply:
It is the parallelized version of AndOr algebra right? It is beneficial to add a comment in docstring to related these two structs.

Reply:
I added some comments in the docstring.